Automattic/wp-calypso
# Design Picker: Add Marketplace Theme to cart

Fixes https://github.com/Automattic/dotcom-forge/issues/2957

## Proposed Changes

Add the selected Marketplace Theme together with the Business plan on the Upgrade Modal.

## Testing Instructions

* Apply the diff D118900-code to your sandbox and direct requests to the sandbox
* Go to the Design Picker page with a Marketplace theme selected `/setup/update-design/designSetup?siteSlug=:site&theme=:theme_slug`. Theme slugs such as `makoney` and `olsen-fse` can be used.
* Click on `Upgrade to activate`
* You should be redirected to the cart with:
  * The business plan
  * The selected product with the annual option (you should have the option to change to monthly)

PS: The modal will not display the proper information yet, it will be handled separately on https://github.com/Automattic/dotcom-forge/issues/2956

## Pre-merge Checklist
